.gitignore

```diff
@@ -84,6 +84,7 @@ t/plugin/grpc-web/grpc-web-server
 t/plugin/grpc-web/node_modules/
 pack
 pack-v*-linux.tgz*
+etcd_data/
```

Comment:
Please ensure that you do not make any unnecessary changes or modifications that are not relevant to the task